src/LudoGame/GameEngine/Class1.cs

```diff
@@ -0,0 +1,90 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace GameEngine
+{
+    public class Game
+    {
+        // Deltagarna ligger här!
+        public List<Player> Players { get; set; }
+
+        public Game(List<Player> players)
+        {
+            Players = players;
+        }
+    }
+
+    public class Player
+    {
+        // Spelarens pjäser ligger här!
+        public List<Piece> Pieces { get; set; }
+        private int number;
+
+        public Player(int number)
+        {
+            Pieces = new List<Piece>(4);
+            this.number = number;
+        }
+
+        public override string ToString() => $"Spelare {number}";
+
+    }
+
+    public enum PieceColor
+    {
+        Red, Blue, Yellow, Green
+    }
+
+    public class Piece
+    {
+        public PieceColor Color { get; set; }
```

Comment:
Are there any specific reason that this and the two following lines are 1) properties 2) public?